test/e2e/features/console/console-autocomplete.test.ts

```diff
@@ -16,12 +15,9 @@ test.describe('Console Autocomplete', {
 	test('Python - Verify Console Autocomplete [C947968]', async function ({ app, python }) {
 		await app.workbench.positronConsole.pasteCodeToConsole('import pandas as pd');
 		await app.workbench.positronConsole.sendEnterKey();
-		await app.workbench.positronConsole.typeToConsole('df = pd.Dat');
 
-		const suggestionList = await app.workbench.positronConsole.getSuggestions();
-		if (suggestionList.length < 3) {
-			fail('Less than 3 suggestions found');
-		}
+		await app.workbench.positronConsole.typeToConsole('df = pd.Dat');
+		await expect(app.workbench.positronConsole.suggestionList).toHaveCount(8, { timeout: 15000 });
```

Comment:
cool.  and this is the same count on windows, too, right?
i could have sworn there were diffs but someone may well have fixed something

Reply:
Yep! https://app.currents.dev/instance/aAPcTxXwztr6oZ6W